caps.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const events = require('./events.js');
+
+require('./vendors/vendor.js');
+require('./drivers/driver.js');
+
+// payload basically means "container object" to hold our data
+events.on('pickup', payload => {
+  const event = { event: 'pickup', time: new Date(), payload: payload};
+  console.log('EVENT', event);
```

Comment:
you can put this as a function that takes in an event and payload. Within the function you can put time and payload in a variable and call the variable in  console.log which will have the 'Event', event, and variable name here a quick example:
```
events.on('delivered', payload => eventlogger('delivered', payload));

function eventlogger(event, payload) {
  let time = {time: new Date(), payload: payload};
  console.log('EVENT', event,time);
  ```